ios/.xcode.env

```diff
@@ -0,0 +1,12 @@
+# This `.xcode.env` file is versioned and is used to source the environment
+# used when running script phases inside Xcode.
+# To customize your local environment, you can create an `.xcode.env.local`
+# file that is not versioned.
+
+# NODE_BINARY variable contains the PATH to the node executable.
+#
+# Customize the NODE_BINARY variable here.
+# For example, to use nvm with brew, add the following line
+# . "$(brew --prefix nvm)/nvm.sh" --no-use
+export NODE_BINARY="${NODE_BINARY:-node}"
```

Comment:
I think we could get this from Nix by just using `which` in a Nix shell:
```
 > time nix-shell -A shells.nodejs --run 'which node' default.nix 2>/dev/null 
/nix/store/zarkkci85li75a3rk1ssalcr3zvpn2j1-nodejs-18.9.1/bin/node

real    3.431s
user    1.924s
sys    0.787s
```
It is a bit slow though because it also calls `nix/scripts/node_modules.sh`.
But we could use a shell without that:
```diff
diff --git a/nix/shells.nix b/nix/shells.nix
index 205892c3f5..56c7388401 100644
--- a/nix/shells.nix
+++ b/nix/shells.nix
@@ -21,6 +21,12 @@ let
 
     nodejs = nodejs-sh;
 
+    # for setting NODE_BINARY in .xcode.env
+    nodejs-only = mkShell {
+      inputsFrom = [ default ];
+      buildInputs = with pkgs; [ node ];
+    };
+
     # for calling clojure targets in CI or Makefile
     clojure = mkShell {
       buildInputs = with pkgs; [ clojure flock maven openjdk ];
```
And then it's faster:
```
 > time nix-shell -A shells.nodejs-only --run 'command -v node' default.nix 2>/dev/null

real    1.354s
user    0.882s
sys    0.256s
```